aws-predictions/src/main/java/com/amplifyframework/predictions/aws/http/LivenessWebSocket.kt

```diff
@@ -76,6 +78,24 @@ internal class LivenessWebSocket(
     private val signer = AWSV4Signer()
     private var credentials: Credentials? = null
 
+    internal var offset = 0L
+    internal enum class ReconnectState {
+        INITIAL,
+        RECONNECTING,
+        RECONNECTING_AGAIN;
```

Comment:
RECONNECTING_AGAIN is unused?

Reply:
it was supposed to be caught by an if statement checking if `== ReconnectState.RECONNECTING`, but it looks like the logic wasn't working.  I'm just adding an if statement in `start()` to make sure